### ISSUE TITLE
Can't `invoke bower`

### ISSUE THREAD

**Issue by lendle** [jupyter/nbviewer#535]
I'm trying to follow the [local installation](https://github.com/jupyter/nbviewer#local-installation) but when I run `invoke bower` I get 
```
/bin/sh: /Users/sam/gitrepos/nbviewer/node_modules/.bin/bower: No such file or directory
```
Similar issue with `invoke less`.

I'm not familiar with `invoke` so I'm not sure how to debug.

**Comment by lendle** [comment]
Running `npm install bower less` before invoke seemed to solve the issue. (I don't know anything about node so I don't know if that's the right way to do things).

I'll leave this issue open because it wasn't clear from the docs that I needed to do this, or if this should have been done automatically, it wasn't.

**Comment by bollwyvl** [comment]
Looks like this is closed by #537!

Thanks, all!

**Comment by lendle** [comment]
Awesome thanks!
